pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -367,7 +380,10 @@ def _format_checkpoint_name(
             metrics.update({"epoch": epoch, 'step': step})
             for group in groups:
                 name = group[1:]
-                filename = filename.replace(group, name + "={" + name)
+
+                if auto_insert_metric_name:
+                    filename = filename.replace(group, name + "={" + name)
```

Comment:
filename = filename.replace('/', '_')

Reply:
Hey @talregev 
Thanks for the comment. 
Could you open an issue about it, describing the need for this?